.eslintrc.json

```diff
@@ -47,7 +47,9 @@
         "prettier/@typescript-eslint"
       ],
       "plugins": ["@typescript-eslint/eslint-plugin"],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
Up for discussion if we want to keep this, I liked not needing to write return type on components mostly.